pkg/components/linkerd/component.go

```diff
@@ -158,6 +159,6 @@ func generateCertificates() (cert, error) {
 		Key:    internal.Indent(root.Cred.EncodePrivateKeyPEM(), 8),
 		Cert:   internal.Indent(root.Cred.Crt.EncodeCertificatePEM(), 8),
 		CA:     internal.Indent(root.Cred.Crt.EncodeCertificatePEM(), 4),
-		Expiry: root.Cred.Crt.Certificate.NotAfter.String(),
+		Expiry: root.Cred.Crt.Certificate.NotAfter.Format(time.RFC3339),
```

Comment:
Is this change backward-compatible?

Reply:
Do you mean will it work with old version of linkerd?

Reply:
Yes. If it's not, we should perhaps commit it together with the update.